server/src/main/java/org/opensearch/snapshots/SnapshotShardsService.java

```diff
@@ -601,6 +601,15 @@ private void notifyFailedSnapshotShard(final Snapshot snapshot, final ShardId sh
         );
     }
 
+    /** Notify the cluster-manager node that the given shard failed to be snapshotted **/
+    private void notifyFailedSnapshotShard(final Snapshot snapshot, final ShardId shardId, final String failure, String generation) {
```

Comment:
The overloaded `notifyFailedSnapshotShard()` method right above this should probably just call this one with a null generation.

Reply:
Good catch. Updated the method above to re-use this method and pass a null generation.